terraform/development/dynamodb.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_dynamodb_table" "contactdetailsapi_dynamodb_table" {
-    name                  = "contact-details"
+    name                  = "Contact Details"

```

Comment:
This needs to be **ContactDetails** (no spaces)

Reply:
I've removed the space